core/src/main/scala/kafka/network/RequestChannel.scala

```diff
@@ -73,14 +73,14 @@ object RequestChannel extends Logging {
     buffer = null
     private val requestLogger = Logger.getLogger("kafka.request.logger")
 
-    private def requestDesc: String = {
+    private def requestDesc(details: Boolean): String = {
```

Comment:
Yes. More likely the ``default`` param could be false and eventually ``true``. But nevermind, I didn't make my mind if it was worth any change. Otherwise, LGTM.